app/components/form/fields/ListboxField.tsx

```diff
@@ -79,6 +79,7 @@ export function ListboxField<
         name={name}
         hasError={fieldState.error !== undefined}
         isLoading={isLoading}
+        buttonRef={field.ref}
```

Comment:
Something I noticed while looking at this is that even after adding `ref`, we are not giving all the props from `field` to `Listbox`. We are still missing `onBlur` and `disabled`, both of which could become more UX-relevant in the future. For example, if we change a form from the RHF default of validating on submit to validating on blur, the lack of `onBlur` here will be a problem. Similarly, if we do #1895, I think the lack of disabled here will be a problem. We can fix it here, but I'm guessing there are a bunch more cases like this.